pkgs/top-level/python-packages.nix

```diff
@@ -2360,6 +2360,8 @@ self: super: with self; {
 
   collidoscope = callPackage ../development/python-modules/collidoscope { };
 
+  color-operations = callPackage ../development/python-modules/color-operations { };
```

Comment:
What about changing the package name to `rio-color-operations` ?

Reply:
> Lightweight version of [rio-color](https://github.com/mapbox/rio-color) but removing rasterio dependency

`color-operations` is not a rasterio plugin.

Reply:
OK. I agree.